Tester/Framework/Environment.php

```diff
@@ -136,8 +136,8 @@ public static function skip($message = '')
 	 */
 	public static function lock($name = '', $path = '')
 	{
-		static $lock;
-		flock($lock = fopen($path . '/lock-' . md5($name), 'w'), LOCK_EX);
+		static $locks = array();
```

Comment:
@MartinMystikJonas Can you remove the initialization? No big deal, it's just shorter.